env/WCOSS2.env

```diff
@@ -20,9 +20,7 @@ export mpmd="--cpu-bind verbose,core cfp"
 export npe_node_max=128
 
 # Configure MPI environment
-export KMP_STACKSIZE=2048M
-export KMP_AFFINITY=scatter
-export MKL_CBWR=AVX2
+#export MKL_CBWR=AVX2
```

Comment:
AVX2 should stay. It has to do with reproducibility: https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-the-conditional-numerical-reproducibility-cnr.html

Reply:
I asked @StevenEarle-NCO about the `export MKL_CBWR=AVX2`, and he recommends to remove it.
This setting is for Intel chips.  WCOSS2 has AMD chips, and this setting is not used/required.

Reply:
It looks to me like it is a compiler option, and we still use Intel compilers and chips with the Intel instruction set (AVX2), even if they are AMD chips. I'm certainly not an expert though and could be wrong.

Reply:
I'm also no expert here either but here's a useful link from Intel:
https://www.intel.com/content/www/us/en/develop/documentation/onemkl-linux-developer-guide/top/obtaining-numerically-reproducible-results/get-started-with-conditional-num-reproducibility.html

In short, the statement should not be used on non-intel CPUs, even if those CPUs support AVX2 instructions. (See NOTE in above link). Even though our AMD chips support AVX2 instructions (you didn't see a failure), it's not the recommended way to go... in fact, MKL should not be used either. When you use "ftn" wrapper to compiler code, it will put optimizations in for our chips so you should not compile anything with AVX2. And cray-libsci is the math library that should be used. Neither MKL nor AVX2 will cause failure, but both will cause performance degradation... amount of degradation will vary depending on code.

Reply:
@StevenEarle-NCO Thanks, that wasn't clear with the docs I originally looked at. I'm not close enough to the code to know if MKL was switched to cray-libsci for all programs on WCOSS2. Might be something to look at as others continue to work on performance.